src/renderer.ts

```diff
@@ -146,7 +145,7 @@ const textElement = (pillar: Pillar) => (node: Node, key: number): ReactNode =>
         case '#text':
             return transform(text, pillar);
         case 'SPAN':
-            return text;
+            return h('span', null, text);
```

Comment:
If a `<span>` has no styles, does it make sense to just simplify it to a text node?

Reply:
Could do, I thought this could break some styles so I'll revert. I just thought that this could be a really annoying bug for anyone expecting to see a span